Server-Monks/nutshell
# Caroline registration and login

# Description

Commented out code for events. Cleaned up main.js

Fixes # (issue)

Resolved merge conflicts with master and current branch.

Please delete options that are not relevant.

- [x ] Bug fix (non-breaking change which fixes an issue)
- [ x] Breaking change (fix or feature that would cause existing functionality to not work as expected)

# Testing Instructions

Run NPM and make sure that everyone's section renders to the dom. Most changes made in main.js with deleting commented out code. Also organized by author.

# Checklist:

- [x ] My code follows the style guidelines of this project
- [x ] I have performed a self-review of my own code
- [x ] I have commented my code, particularly in hard-to-understand areas
- [x ] My changes generate no new warnings or errors
- [x ] I have added test instructions that prove my fix is effective or that my feature works
